src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -31,10 +31,10 @@ class BrandedCheckoutStep1Controller {
 
   initItemConfig () {
     this.itemConfig = {}
-    this.itemConfig.campaign_code = this.campaignCode
-    if (this.itemConfig.campaign_code &&
-      (this.itemConfig.campaign_code.match(/^[a-z0-9]+$/i) === null || this.itemConfig.campaign_code.length > 30)) {
-      this.itemConfig.campaign_code = ''
+    this.itemConfig.CAMPAIGN_CODE = this.campaignCode
+    if (this.itemConfig.CAMPAIGN_CODE &&
+      (this.itemConfig.CAMPAIGN_CODE.match(/^[a-z0-9]+$/i) === null || this.itemConfig.CAMPAIGN_CODE.length > 30)) {
```

Comment:
just to clarify if the regular expression should or shouldn't match

Reply:
This logic isn't changing, just went from lowercase to uppercase.